packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/pools/_components/addEditConnectionPool.tsx

```diff
@@ -0,0 +1,342 @@
+                        {databases.map((option) => (
+                          <SelectItem key={option.id} value={option.id}>
+                            {option.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="mode"
+              render={({ field }) => (
+                <FormItem className="flex flex-col gap-1 mt-2">
+                  <FormLabel>{t('formConnectionPoolFieldModeLabel')}</FormLabel>
+                  <FormControl>
+                    <Select value={field.value} onValueChange={field.onChange}>
+                      <SelectTrigger>
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {Object.values(
+                          database.postgresql.connectionpool.ModeEnum,
+                        ).map((option, index) => (
+                          <SelectItem key={index} value={option}>
+                            {option}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="size"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>{t('formUserFieldSizeLabel')}</FormLabel>
+                  <FormControl>
+                    <SizeInput
+                      value={field.value}
+                      onChange={(newValue) => form.setValue('size', newValue)}
+                      min={POOL_CONFIG.size.min}
+                      max={POOL_CONFIG.size.max}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+
+            <FormField
+              control={form.control}
+              name="userId"
+              render={({ field }) => (
+                <FormItem className="flex flex-col gap-1 mt-2">
+                  <FormLabel>{t('formConnectionPoolFieldUserLabel')}</FormLabel>
+                  <FormControl>
+                    <Select value={field.value} onValueChange={field.onChange}>
+                      <SelectTrigger>
+                        <SelectValue
+                          placeholder={t(
+                            'formConnectionPoolFieldUserPlaceholder',
+                          )}
+                        />
+                      </SelectTrigger>
+                      <SelectContent>
+                        <SelectItem key={1} value="-">
```

Comment:
Have you tried removing the value or setting it to undefined ? It should give an undefined value to the field, which looks like what you want here

Reply:
yep you're right! Changed it.